datahub-web-react/src/app/shared/avatar/CustomAvatar.tsx

```diff
@@ -64,8 +68,21 @@ export default function CustomAvatar({
     if (!name) {
         return url ? <Link to={url}>{avatar}</Link> : avatar;
     }
+
+    const renderTitle = (input) => {
+        let title = `${input}`;
+        if (isGroup) {
+            title = `${title} - Group`;
+        } else if (isPolicy) {
+            title = `${title}`;
```

Comment:
Hey this is a serious followup thing:

I don't really think we should use an Avatar to represent a Policy (or a Role for that matter). 

Instead, we should simply show a Clickable link which is the name of the Policy / Role.

Reply:
i agree, I can add a to-do here